app/routes/organen/orgaan/mandatarissen.js

```diff
@@ -161,12 +161,13 @@ export default class OrganenMandatarissenRoute extends Route {
   }
 
   getOptions(params, bestuursOrgaan) {
+    const NO_PAGINTATION = {
+      number: params.page,
+      size: 9999,
+    };
```

Comment:
Thought that this was the clearest way to say that it does not have pagination

Reply:
Maybe we also should set the page number to always be 0?

Reply:
Small typo here as well.

Reply:
Removed it in this route